public/app/features/variables/editor/VariableEditorEditor.tsx

```diff
@@ -149,8 +149,10 @@ export class VariableEditorEditorUnConnected extends PureComponent<Props> {
                   onChange={this.onNameChange}
                   name="Name"
                   placeholder="name"
-                  required
                   testId={selectors.pages.Dashboard.Settings.Variables.Edit.General.generalNameInputV2}
+                  maxLength={VariableNameConstraints.MaxSize}
+                  required
+                  tooltip="Variable name cannot be bigger than 50 characters"
```

Comment:
```suggestion
                  tooltip="Variable name cannot be longer than 50 characters"
```
Just personal preference 😋

Reply:
Thank you! 🤗